### PR TITLE
feat(cli): return os error status code 1 when siren cli return error

### LINKED ISSUE #6: Error OS status code on Siren CLI
**Is your feature request related to a problem? Please describe.**
Siren CLI is currently does not gracefully return os status code (it is always returning 0). We need to return os status code != 0 when Siren CLI return error.

**Describe the solution you'd like**
Assign `os.Exit(1)` when error in `main.go`.

### PR DESCRIPTION
**Pull request by mabdh**
close #6

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -14,6 +14,7 @@ func main() {
 	defer stop()
 
 	if err := cli.New().ExecuteContext(ctx); err != nil {
-		fmt.Printf("%+v\n", err)
+		fmt.Fprintf(os.Stderr, "\n%v\n", err)
+		os.Exit(1)
```
Comment (StewartJingga) [inline main.go:18]: `os.Exit` skips `defer`.

we might want to restructure call order or add wrap it into another function.
ref: https://github.com/golang/go/issues/38261

**Review by StewartJingga — CHANGES_REQUESTED** [review]

File: main.go (hunk 1 above)
Comment (mabdh) [reply main.go:18]: I think it is fine for our case. The line `ctx, stop := signal.NotifyContext(context.Background(), os.Interrupt)` is just to handle interrupt with keyboard.

I have checked the behaviour and it also return status code 1.

Comment (mabdh) [reply main.go]: fixed stew

**Review by StewartJingga — APPROVED** [review]
LGTM

**Review by rahmatrhd — APPROVED** [review]

**Pull request merged**